adapters/handlers/grpc/v1/prepare_reply.go

```diff
@@ -34,7 +34,8 @@ import (
 func searchResultsToProto(res []interface{}, start time.Time, searchParams dto.GetParams, scheme schema.Schema, usesPropertiesMessage bool) (*pb.SearchReply, error) {
 	tookSeconds := float64(time.Since(start)) / float64(time.Second)
 	out := &pb.SearchReply{
-		Took: float32(tookSeconds),
+		Took:                    float32(tookSeconds),
+		GenerativeGroupedResult: new(string), // pointer to empty string
```

Comment:
why is this needed? Can't we just leave it `nil`?

Reply:
Not having it broke a unit the `group by` unit test in `prepare_reply_test.go` on L976 due to a nil pointer deference

In addition, gRPC field is `optional string` and the previous behaviour was for it to always be `""` so changing this would be a BC change I think